v7/latex/latex/plugins/latex_formula_image_renderer.py

```diff
@@ -89,6 +89,7 @@ def __init__(self):
         super(LatexImageFormulaRenderer, self).__init__()
         self.__formula_scale = 1.25
```

Comment:
Because they look ugly, because they don't really give privacy, or because of something else?